File: app/src/main/java/com/leilei/guoshujinfu/mylearning/model/req/PictureReqBean.java
package com.leilei.guoshujinfu.mylearning.model.req;

/**
 * @Author: AqrLei
 * @Name MyLearning
 * @Description:
 * @Date: 2017/8/4
 */

public final class PictureReqBean {
    private String type;



    public PictureReqBean() {

    }

    public PictureReqBean(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: truckrental/src/main/kotlin/com/aqrlei/graduation/truckrental/model/req/PictureReqBean.kt
package com.aqrlei.graduation.truckrental.model.req

/**
 * @Author: AqrLei
 * @Name MyLearning
 * @Description:
 * @Date: 2017/8/4
 */

/*request bean*/
class PictureReqBean {
    var type: String? = null

    constructor() {

    }

    constructor(type: String) {
        this.type = type
    }
}
